tests/irepela/homework_5/test_sets.py

```diff
@@ -0,0 +1,27 @@
+some_set = {"a", "b", "c"}
+some_set_2 = {"b", "a", "c"}
+
+
+def test_sets_compare():
+    assert some_set == some_set_2
+    assert some_set is not some_set_2
+    assert {"a"} < some_set
+
+
+def test_subset_superset():
+    assert {"a", "c"}.issubset(some_set)
```

Comment:
assert {"a", "b", "c"}.issubset(some_set)
assert {"a", "b", "c"}  <= some_set
assert not {"a", "b", "c"}  < some_set